temporalcli/commandsmd/commands.md

```diff
@@ -338,8 +338,7 @@ Includes options set for [payload input](#options-set-for-payload-input).
 * `--run-id`, `-r` (string) - Run Id. Cannot be set when query is set.
 * `--query`, `-q` (string) - Start a batch to Signal Workflow Executions with given List Filter. Either this or
   Workflow Id must be set.
-* `--reason` (string) - Reason to perform batch. Only allowed if query is present. Defaults to message with user name
-  and time.
+* `--reason` (string) - Reason to perform batch. Only allowed if query is present unless the command specifies otherwise. 
```

Comment:
I think reason needs to be moved out of here and added individually for every command.
I tried phrasing it generically but it comes out unhelpful.

Reply:
Terminate seems to be the only command that needs it (from a brief grep of our protos), so I'm just going to break it out for this single command. All other batch ops can use this

Reply:
schedule pause/unpause also takes a reason. It could be a different flag though.